discovery-provider/plugins/network-monitoring/src/metrics/queries.ts

```diff
@@ -206,3 +206,21 @@ export const getUnsyncedUsersCount = async (run_id: number): Promise<number> =>
     return usersCount
 }
 
+// The number of users whose primary content node is null
+export const getGhostUsersCount = async (run_id: number): Promise<number> => {
+    const usersResp: unknown[] = await sequelizeConn.query(`
+        SELECT COUNT(*) as user_count
+        FROM network_monitoring_users
+        WHERE 
+            run_id = :run_id
+        AND 
+            primary_clock_value IS NULL;
```

Comment:
this query does not seem to match description/intent? is it that primary has no clock value or that primarySpID is null?